src/pages/home/report/ReportActionCompose/AttachmentPickerWithMenuItems.tsx

```diff
@@ -401,7 +401,11 @@ function AttachmentPickerWithMenuItems({
                                     });
                                 }
                             }}
-                            anchorPosition={styles.createMenuPositionReportActionCompose(shouldUseNarrowLayout, windowHeight, windowWidth)}
+                            anchorPosition={
+                                canUseLeftHandBar
+                                    ? styles.createMenuPositionReportActionComposeWhenLhbIsVisible(shouldUseNarrowLayout, windowHeight, windowWidth)
```

Comment:
NAB: The shortcuts in names should be uppercase
```suggestion
                                    ? styles.createMenuPositionReportActionComposeWhenLHBIsVisible(shouldUseNarrowLayout, windowHeight, windowWidth)
```